test/integration/openid4ci_test.go

```diff
@@ -123,6 +123,14 @@ func TestOpenID4CIFullFlow(t *testing.T) {
 			subID := getDisclosure(t, serializedVC, "id")
 			require.Contains(t, subID, didID)
 		}
+
+		displayData, err := interaction.ResolveDisplay("en-US")
+		require.NoError(t, err)
+
+		println("displayData:", string(displayData.Data))
+
+		require.Contains(t, string(displayData.Data), "john.doe@foo.bar")
```

Comment:
Can we validate the exact json key field instead of generic `contains` on the whole string ? This will help remove false positives.

Reply:
we can do it after merging of https://github.com/trustbloc/wallet-sdk/pull/224 not to do double work